### PR TITLE
SIG UI Annual Report 2022

### LINKED ISSUE #7118: 2022 Annual Report: SIG UI
Chairs: @floreks @maciaszczykm @shu-mutou 
Liaison: @justaugustus

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-ui/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-ui/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig ui
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by shu-mutou**
Annual report 2022 for SIG UI.

**Which issue(s) this PR fixes**:

Fixes #7118

### CODE REVIEW

**Comment by shu-mutou** [comment]
/retitle [WIP] SIG UI Annual Report 2022

File: sig-ui/annual-report-2022.md [hunk 1]
```diff
@@ -4,93 +4,96 @@
...
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We have translation sub teams in Kubernetes Dashboard that have responsibility to manage translation files for each languages. And we have [special guide](https://github.com/kubernetes/dashboard/blob/master/docs/developer/internationalization.md) to organize and to manage the translation sub teams.
+   - Also, the development environment container is available to facilitate translation work.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, current chairs are spread between Kubermatic and NEC. And contributors are also spread across multiple companies including Wiremind, PWC and Tencent. See more [contributors in 2021](https://github.com/kubernetes/dashboard/graphs/contributors?from=2022-01-01&to=2022-12-31&type=c).
```
Comment (maciaszczykm) [inline sig-ui/annual-report-2022.md:54]: s/Kubermatic/Plural

**Comment by shu-mutou** [comment]
@maciaszczykm @floreks Do you have any other topics or comments?

**Comment by maciaszczykm** [comment]
No, looks good to me.

**Comment by shu-mutou** [comment]
@justaugustus PTAL

**Comment by shu-mutou** [comment]
@justaugustus PTAL

**Review by justaugustus — APPROVED** [review]
Thanks @shu-mutou!!

**Pull request merged**